packages/flame_test/pubspec.yaml

```diff
@@ -15,6 +15,7 @@ dependencies:
     sdk: flutter
   meta: ^1.7.0
   test: ^1.17.12
+  vector_math: ^2.1.2
```

Comment:
Shouldn't this be `any` too?

Reply:
we can't have published packages on pub that have a dep with any on its pubspec, so I added the last one

Reply:
Alright, seems like it is too new and that we have to use `2.1.1` then?

Reply:
yeah, which is weird because it works on my machine, but either way I am downgrading it and upload a commit